proto/api/v1/credential_expiry_service.proto

```diff
@@ -12,9 +12,12 @@ package v1;
 
 message GetCertExpiry {
     enum Component {
-        UNKNOWN = 0;
-        CENTRAL = 1;
-        SCANNER = 2;
+        UNKNOWN            = 0;
+        CENTRAL            = 1;
+        SCANNER            = 2;
+        SCANNER_V4_INDEXER = 3;
+        SCANNER_V4_MATCHER = 4;
+        SCANNER_V4_DB      = 5;
```

Comment:
From the service API perspective we probably just need the `SCANNER_V4` component (or bundle it all together with the existing `SCANNER` component if that makes sense).

Scanner + ScannerDB are bundled into one `Component`. In the UI there is only one action to take for `Scanner` today:

![image](https://github.com/stackrox/stackrox/assets/119438707/52c4090d-fc06-4a67-85ca-bad1a06c6fca)

The `yaml` contains secrets for `scanner-tls` and `scanner-db-tls`, think it would makes sense that the cred expiry service will report on the 'soonest to expire' cert of `scanner-v4-indexer`, `scanner-v4-matcher`, and `scanner-v4-db`.    When the `yaml` is applied, it will refresh the certs for all the `scanner-v4` components.

Reply:
I have simplified this: now the only addition here is `SCANNER_V4`.